easybuild/easyblocks/generic/cmakemake.py

```diff
@@ -134,12 +134,18 @@ def configure_step(self, srcdir=None, builddir=None):
 
         options_string = ' '.join(options)
 
-        command = ' '.join([
-            self.cfg['preconfigopts'],
-            self.cfg.get('configure_cmd') or DEFAULT_CONFIGURE_CMD,
-            options_string,
-            self.cfg['configopts'],
-            srcdir])
+        if self.cfg.get('configure_cmd') != DEFAULT_CONFIGURE_CMD:
```

Comment:
@migueldiascosta Do you mind changing this to positive logic, i.e.:

```python
if self.cfg.get('configure_cmd') == DEFAULT_CONFIGURE_CMD:
    ...
else:
    ...
```

Reply:
will do